scripts/microbe_stage/organelle.lua

```diff
@@ -260,6 +263,10 @@ function OrganelleFactory.makeOrganelle(data)
     end
     local success, organelle = pcall(make_organelle)
```

Comment:
when input data to OrganelleFactory:makeOrganelle doesn't have a good name, the function it tried to call does not exist